installed linux binary for fedora + hyprland

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x859f0e]

goroutine 1 [running]:
github.com/ayn2op/discordo/cmd.(*application).showModal(0x0, {0xa0db3e, 0x9}, {0xc0000f4d80, 0x213}, {0xc000207948, 0x2, 0x2}, 0xc000286180)
	/home/runner/work/discordo/discordo/cmd/application.go:214 +0x4e
github.com/ayn2op/discordo/cmd.(*application).showErrorModal(0x0, {0xa16ac7, 0x18}, {0xc00025a400, 0x1f2}, {0x0, 0x0, 0x0})
	/home/runner/work/discordo/discordo/cmd/application.go:245 +0x16f
github.com/ayn2op/discordo/cmd.(*application).onError(0x0, {0xa16ac7, 0x18}, {0xad5d00, 0xc00028a440}, {0x0, 0x0, 0x0})
	/home/runner/work/discordo/discordo/cmd/application.go:210 +0x1f9
github.com/ayn2op/discordo/cmd.newMessageInput(0xc0000c9108)
	/home/runner/work/discordo/discordo/cmd/message_input.go:62 +0x19d
github.com/ayn2op/discordo/cmd.newApplication(0xc0000c9108)
	/home/runner/work/discordo/discordo/cmd/application.go:44 +0x177
github.com/ayn2op/discordo/cmd.init.func1(0xc0001f6400?, {0xa0b607?, 0x4?, 0xa0b3b7?})
	/home/runner/work/discordo/discordo/cmd/root.go:67 +0x258
github.com/spf13/cobra.(*Command).execute(0xe1cdc0, {0xc0000240a0, 0x0, 0x0})
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.10.1/command.go:1015 +0xb02
github.com/spf13/cobra.(*Command).ExecuteC(0xe1cdc0)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.10.1/command.go:1148 +0x465
github.com/spf13/cobra.(*Command).Execute(...)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.10.1/command.go:1071
main.main()
	/home/runner/work/discordo/discordo/main.go:10 +0x1e

--- a/cmd/application.go
+++ b/cmd/application.go
@@ -1,9 +1,9 @@
 package cmd
 
 import (
+	"fmt"
 	"log/slog"
 	"strings"
-	"fmt"
 
 	"github.com/ayn2op/discordo/internal/config"
 	"github.com/ayn2op/discordo/internal/keyring"
@@ -42,6 +42,10 @@ func newApplication(cfg *config.Config) *application {
 		guildsTree:   newGuildsTree(cfg),
 		messagesList: newMessagesList(cfg),
 		messageInput: newMessageInput(cfg),
+	}
+
+	if err := clipboard.Init(); err != nil {
+		app.onError("Failed to init clipboard", err)
 	}
 
 	app.pages.SetInputCapture(app.onPagesInputCapture)
@@ -242,7 +246,7 @@ func (a *application) showConfirmModal(prompt string, buttons []string, onDone f
 }
 
 func (a *application) showErrorModal(msg, err string, info ...any) {
-	a.showModal("[ ERROR ]", msg + "\nReason: " + err, []string{"Copy", "OK"}, func(label string) {
+	a.showModal("[ ERROR ]", msg+"\nReason: "+err, []string{"Copy", "OK"}, func(label string) {
 		if label != "Copy" {
 			return
 		}

--- a/cmd/message_input.go
+++ b/cmd/message_input.go
@@ -58,17 +58,13 @@ func newMessageInput(cfg *config.Config) *messageInput {
 		mentionsList:    tview.NewList(),
 	}
 
-	if err := clipboard.Init(); err != nil {
-		app.onError("Failed to init clipboard", err)
-	} else {
-		mi.
-			SetClipboard(func(s string) {
-				clipboard.Write(clipboard.FmtText, []byte(s))
-			}, func() string {
-				data := clipboard.Read(clipboard.FmtText)
-				return string(data)
-			})
-	}
+	mi.
+		SetClipboard(func(s string) {
+			clipboard.Write(clipboard.FmtText, []byte(s))
+		}, func() string {
+			data := clipboard.Read(clipboard.FmtText)
+			return string(data)
+		})
 
 	mi.Box = ui.ConfigureBox(mi.Box, &cfg.Theme)
 	mi.SetInputCapture(mi.onInputCapture)

